scripts/manage_users.py

```diff
@@ -0,0 +1,117 @@
+                        action='store_true', help='Make user a superuser')
+    parser.add_argument('-u', '--usual-user', required=False,
+                        action='store_true', help='Make user a usual one')
+    return parser.parse_args()
+
+
+async def main() -> int:
+    arguments = parse_args()
+    async for db in get_db():
+        async with db.begin():
+            user = (await db.execute(select(models.User).where(
+                models.User.email == arguments.email).options(
+                selectinload(models.User.roles),
+                selectinload(models.User.oauth_accounts),
+            ))).scalars().first()
+            if not user:
+                raise ValueError(f'No such user in the system: '
+                                 f'{arguments.email}')
+
+            if arguments.verify and arguments.deactivate:
+                raise ValueError('Cannot both activate and deactivate user')
+
+            if arguments.superuser and arguments.usual_user:
+                raise ValueError('Cannot both make user a superuser '
+                                 'and usual one')
+
+            if arguments.verify:
+                await user_crud.activate_user(user.id, db)
+
+            if arguments.deactivate:
+                await user_crud.deactivate_user(user.id, db)
+
+            if arguments.superuser:
+                await user_crud.make_superuser(user.id, db)
+
+            if arguments.usual_user:
+                await user_crud.make_usual_user(user.id, db)
+
+            if (arguments.add_roles or
+                    arguments.remove_roles) and not arguments.team_name:
+                print('Cannot assign roles without team specified, exiting')
+                return 1
+
+            team = (await db.execute(select(models.Team).where(
+                models.Team.name == arguments.team_name))).scalars().first()
+            if not team:
+                raise ValueError(f'No such team in the system: '
+                                 f'{arguments.team_name}')
+
+            add_roles = None
+            remove_roles = None
+
+            if arguments.add_roles:
+                add_roles = (await db.execute(select(models.UserRole).where(
+                    models.UserRole.name.in_(
+                        [team_crud.get_team_role_name(team.name, r)
+                         for r in arguments.add_roles]))
+                )).scalars().all()
+
+            if arguments.remove_roles:
+                remove_roles = (await db.execute(select(models.UserRole).where(
+                    models.UserRole.name.in_(
+                        [team_crud.get_team_role_name(team.name, r)
+                         for r in arguments.remove_roles]))
+                )).scalars().all()
+
+            if add_roles:
+                user.roles.extend(add_roles)
+            if remove_roles:
+                roles_ids = [r.id for r in remove_roles]
+                await db.execute(delete(models.UserRoleMapping).where(
```

Comment:
can we delete user roles through `user.roles.remove(role)`?

Reply:
Unfortunately it doesn't work that way because of many-to-many relationship